### ISSUE TITLE
[Feature:UI] UI Management for Environments

### ISSUE THREAD

**Issue by CESARDELATORRE** [danielsc/azureml-workshop-2019#22]
There's no UI Management for Environments. Since Environments are infrastructure assets that can be re-used (in many ways comparable to Model management or Compute management) there should be a UO feature to manage Environments such as importing from Conda, Cloning from curated environments, list Environments in your Workspace, etc.

**Comment by danielsc** [comment]
agree: https://msdata.visualstudio.com/Vienna/_workitems/edit/583749
